rgit.rb

```diff
@@ -122,11 +146,121 @@ def add(*pathspecs)
     puts("Initialized empty rGit repository in #{directory}") unless options[:quiet]
   end
+ 
+
+  @clone_descriptions = {
+    local:
+      "When the repository to clone from is on a local machine, this flag
+       bypasses the normal \"Git aware\" transport mechanism and clones the
+       repository by making a copy of HEAD and everything under objects and
+       refs directories. The files under .git/objects/ directory are hardlinked
+       to save space when possible.
+
+       If the repository is specified as a local path (e.g., /path/to/repo),
+       this is the default, and --local is essentially a no-op. If the repository
+       is specified as a URL, then this flag is ignored (and we never use the local
+       optimizations). Specifying --no-local will override the default when
+       /path/to/repo is given, using the regular Git transport instead.",
+    bare:
+      ""
+
+
+
+  }
+
+  desc "clone", "Clone a repository into a new directory"
+  long_desc @clone_descriptions[:long_desc]
+
+  method_option :local,
+    { aliases: "l",
+      type: :boolean,
+      desc: @clone_descriptions[:local],
+    }
+  method_option :no_hardlinks,
+    { type: :boolean, 
+      desc: @clone_descriptions[:no_hardlinks],
+    }
+  method_option :shared,
+    { type: :boolean,
+      desc: @clone_descriptions[:shared],
+    }
+  method_option :reference,
+    { type: :string,
+      desc: @clone_descriptions[:reference],
+    }
+  method_option :dissociate,
+    { type: :boolean,
+      desc: @clone_descriptions[:dissociate],
+    }
+  method_option :quiet,
+    { aliases: "q",
+      type: :boolean,
+      desc: @clone_descriptions[:quiet],
+    }
+  method_option :verbose,
+    { aliases: "v",
+      type: :boolean,
+      desc: @clone_descriptions[:verbose],
+    }
+  method_option :progress,
+    { type: :boolean,
+      desc: @clone_descriptions[:progress],
+    }
+  method_option :no_checkout,
+    { aliases: "n",
+      type: :boolean,
+      desc: @clone_descriptions[:no_checkout],
+    }
+  method_option :bare,
+    { type: :boolean,
+      desc: @clone_descriptions[:bare],
+    }
+  method_option :mirror,
+    { type: :boolean,
+      desc: @clone_descriptions[:mirror],
+    }
+  method_option :branch,
+    { aliases: "b",
+      type: :string,
+      desc: @clone_descriptions[:branch],
+    }
+  method_option :upload_pack,
+    { aliases: "u",
+      type: :boolean,
+      desc: @clone_descriptions[:upload_pack],
+    }
+  method_option :template,
+    { type: :string,
+      desc: @clone_descriptions[:template],
+    }
+  method_option :config,
+    { aliases: "c",
+      type: :hash,
+      desc: @clone_descriptions[:config],
+    }
+  method_option :depth,
+    { type: :numeric,
+      desc: @clone_descriptions[:depth],
+    }
+  method_option :single_branch,
+    { type: :boolean,
+      desc: @clone_descriptions[:single_branch],
+    }
+  method_option :separate_git_dir,
+    { type: :string,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.